github-graphql/PullRequestsOpen.gql

```diff
@@ -0,0 +1,26 @@
+query PullRequestsOpen ($repo_owner: String!, $repo_name: String!, $after: String) {
+  repository(owner: $repo_owner, name: $repo_name) {
+    pullRequests(first: 100, after: $after, states:OPEN, labels: ["S-waiting-on-review","T-compiler"]) {
```

Comment:
This should not be limited to these labels.

Reply:
hmm you're right, we should pull **all** open PRs here. Don't know what I was thinking...